### PR TITLE
[test]: hello_world testing code.

### PR DESCRIPTION
**Pull request by wangzhongren-code**
## What?
* brief:  这是一个测试helloworld程序
* author: wangzhongren
## What changes are included in this PR?
add the helloworld testing code
## Are these changes tested?
the code is in the helloworld.cpp

### CODE REVIEW

**Comment by Light-City** [comment]
HelloWord shuold follow lowercase underscore style, such as hello_world or helloworld

File: stl/test/wangzhongren/src/helloworld.cpp [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * @LastEditor: wangzhongren
+ * @LastEditTime: 2023-06-17 17:24:48
+ * @Discription: the testing code for helloword!
+ */
+
+#include <iostream>
+using namespace std;
+ 
+int main() 
+{
```
Comment (Light-City) [inline stl/test/wangzhongren/src/helloworld.cpp:14]: int main() {

}

Comment (wangzhongren-code) [reply stl/test/wangzhongren/src/helloworld.cpp]: ok, I have changed.

**Comment by Light-City** [comment]
LGTM

**Pull request closed without merging**